workflow-runtime/src/main/java/com/squareup/workflow/internal/WorkflowNode.kt

```diff
@@ -279,11 +277,16 @@ internal class WorkflowNode<PropsT, StateT, OutputT : Any, RenderingT>(
    * Applies [action] to this workflow's [state] and
    * [emits an output to its parent][emitOutputToParent] if necessary.
    */
-  private fun <T : Any> applyAction(action: WorkflowAction<StateT, OutputT>): T? {
-    val (newState, output) = action.applyTo(state)
+  private fun <T> applyAction(action: WorkflowAction<StateT, OutputT>): T {
+//    return action.applyTo(state) { newState, hasOutput, output ->
```

Comment:
Did you want to delete this?

Reply:
Yes, once we've figured out how we want this to look.